deploy/datasource.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
```

Comment:
We have other important datasources, see https://github.com/infrawatch/service-telemetry-operator/blob/master/tests/performance-test/grafana/datasource.yaml

Reply:
Actually, the ES datasource is more complicated than what's shown. Specifically notice that it names a single ES index (and it's NOT the one with our events): https://github.com/infrawatch/service-telemetry-operator/blob/master/tests/performance-test/grafana/datasource.yaml#L9

What actually needs to happen somehow is a datasource for EACH plugin since an index is generated for each.

I'm going to say this is a problem for another PR.

Reply:
Yea, this was just an initial import to make sure everything worked. I definitely want to follow up on this and get all our dashboarding into a single area with all the appropriate sources etc. I'd like to do that in subsequent changes.

I can add them here though if you'd like to see everything land in a single commit. I'm good either way.

Reply:
OK cool, makes sense to me as well.